awips_install.sh

```diff
@@ -145,28 +161,49 @@ function check_remove_edex {
 
 function remove_edex {
   while true; do
-    read -p "`echo $'\n'`We are going to back up some files. What location do you want your files backed up to? `echo $'\n> '`" backup_dir
-    if [ ! -d $backup_dir ]; then
+    read -p "`echo $'\n'`We want to back up some configuration files. What location do you want your files backed up to?
+        If you choose not to back up files (you will lose all your configurations) type \"no\"`echo $'\n> '`" backup_dir
+
+    backup_dir=$(echo $backup_dir | tr '[:upper:]' '[:lower:]')
+    if [ $backup_dir = "no" ] || [ $backup_dir = "n" ]; then
+        while true; do
```

Comment:
For this while loop, there is no way to break out of it if the user doesn't confirm? (like if they changed their mind and do want to back up the files?)

Reply:
Technically the user can control+C and exit out of the script, but I could add another line where a user could type "q" or "quit" to exit out of the script completely and then they could restart the script.

Reply:
Hmm... yeah I think something else would be good instead of just sitting in an infinite loop until the user manually kills the script?